extensions/amp-sidebar/0.1/test/test-amp-sidebar.js

```diff
@@ -167,6 +167,9 @@ describes.realWin('amp-sidebar 0.1 version', {
             callback();
           },
         };
+        impl.getHistory_ = () => ({
```

Comment:
nit: You should use the stub format `sandbox.stub(impl, 'getHistory_').returns(/*...*/)` as a defensive programming practice, since the sandbox created by `describes.js` restores stubbed values when it calls `sandbox.restore` in `teardown`. This can help prevent tests from accidentally mutating state between cases, helping keep tests hermetic. It's also a more common pattern than simply reassigning the property, and better communicates your intent.

Reply:
Adding @jridgewell, who is rewriting how tests use sandboxes.